filament/src/details/Renderer.h

```diff
@@ -140,7 +140,11 @@ class FRenderer : public Renderer {
     backend::TextureFormat getLdrFormat(bool translucent) const noexcept;
 
     Epoch getUserEpoch() const { return mUserEpoch; }
-    duration getUserTime() const noexcept { return clock::now() - getUserEpoch(); }
+    double getUserTime() const noexcept {
+        duration d = clock::now() - getUserEpoch();
+        // convert the duration (whatever it is) to a duration in seconds encoded as double
+        return std::chrono::duration_cast<std::chrono::duration<double>>(d).count();
```

Comment:
I'm not sure you need `std::chrono::duration_cast` here.

Reply:
done!

Reply:
done!